cardemo/src/main/resources/static/js/build.js

```diff
@@ -236,7 +250,62 @@ var MyGame = {};
         this.helpButton.inputEnabled = true;
       }, this);
       
-      this.apiClient = new APIclient();
+      var carId = generateID();
+
+        var hostname = "tcp.apps.pcfdemo.solacemessaging.net";
```

Comment:
We really need to figure out a way to make this more portable such that we dont have to change the code each time the tcp routes change.